elections.md

```diff
@@ -127,19 +127,7 @@ History of election officers:
 
 ### Vacancies
 
-In the event of a resignation or other loss of a [bootstrap committee member],
-the position will not be refilled.
```

Comment:
what was the rationale earlier for why it will not be filled ?

Reply:
The bootstrap committee was 12 (13?) seats (extra big to be extra sure about initial set of decisions) and the goal was to get it down to 7 via attrition.

Reply:
Lol
> was to get it down to 7 via attrition.